docs/JLBP-14.md

```diff
@@ -1,7 +1,7 @@
-# [JLBP-14] Do not use version ranges
+# [JLBP-14] Specify exactly one version of each dependency
```

Comment:
I support the general idea of using affirmative statements instead of negated ones, but this particular affirmative statement doesn't feel right. To someone who is used to seeing only single version dependencies used, this is very confusing. I feel like "range" needs to be used somehow, because that's the poor practice.

Reply:
You're right that we rarely see ranges in practice. Given that, I think it's more confusing to put ranges, something many developers do not encounter, in the title.

Reply:
But avoiding ranges is the entire point of this JLBP. Someone who does not encounter ranges  and wouldn't think of using them doesn't even need to think about this JLBP.

Reply:
How about "Avoid version ranges"? (not using negated sentence)

I agree with Garrett. This JLBP is about version ranges; it's clear that having it in the title.